From 376b6b462ae23c851eaf04976c687c540252d58f Mon Sep 17 00:00:00 2001
From: Victor Bo <bvoid@yandex.ru>
Date: Fri, 22 May 2020 12:56:16 +0300
Subject: [PATCH 150/186] rework main fonts (and others) v217

- remove GoogleSans (none-free)
- remove SimpleDay (no info)
- come back stock Roboto as default
---
 data/fonts/Android.mk                         |  4 +-
 data/fonts/fonts.xml                          | 71 ++++++-------------
 packages/overlays/Android.mk                  |  3 +-
 .../overlays/FontRobotoOverlay/Android.mk     | 31 --------
 .../FontRobotoOverlay/AndroidManifest.xml     | 31 --------
 .../FontRobotoOverlay/res/values/config.xml   | 27 -------
 .../FontRobotoOverlay/res/values/strings.xml  | 20 ------
 .../overlays/FontSimpleDayOverlay/Android.mk  | 31 --------
 .../FontSimpleDayOverlay/AndroidManifest.xml  | 31 --------
 .../res/values/config.xml                     | 27 -------
 .../res/values/strings.xml                    | 20 ------
 .../overlays/FontSimpledayOverlay/Android.mk  | 31 --------
 .../FontSimpledayOverlay/AndroidManifest.xml  | 31 --------
 .../res/values/config.xml                     | 27 -------
 .../res/values/strings.xml                    | 20 ------
 15 files changed, 25 insertions(+), 380 deletions(-)
 delete mode 100644 packages/overlays/FontRobotoOverlay/Android.mk
 delete mode 100644 packages/overlays/FontRobotoOverlay/AndroidManifest.xml
 delete mode 100644 packages/overlays/FontRobotoOverlay/res/values/config.xml
 delete mode 100644 packages/overlays/FontRobotoOverlay/res/values/strings.xml
 delete mode 100644 packages/overlays/FontSimpleDayOverlay/Android.mk
 delete mode 100644 packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml
 delete mode 100644 packages/overlays/FontSimpleDayOverlay/res/values/config.xml
 delete mode 100644 packages/overlays/FontSimpleDayOverlay/res/values/strings.xml
 delete mode 100644 packages/overlays/FontSimpledayOverlay/Android.mk
 delete mode 100644 packages/overlays/FontSimpledayOverlay/AndroidManifest.xml
 delete mode 100644 packages/overlays/FontSimpledayOverlay/res/values/config.xml
 delete mode 100644 packages/overlays/FontSimpledayOverlay/res/values/strings.xml

diff --git a/data/fonts/Android.mk b/data/fonts/Android.mk
index f2166e3100e..454dceb9c82 100644
--- a/data/fonts/Android.mk
+++ b/data/fonts/Android.mk
@@ -35,8 +35,8 @@ endef
 ##########################################
 # The following fonts are just symlinks, for backward compatibility.
 ##########################################
-$(eval $(call create-font-symlink,DroidSans.ttf,GoogleSans-Regular.ttf))
-$(eval $(call create-font-symlink,DroidSans-Bold.ttf,GoogleSans-Bold.ttf))
+$(eval $(call create-font-symlink,DroidSans.ttf,Roboto-Regular.ttf))
+$(eval $(call create-font-symlink,DroidSans-Bold.ttf,Roboto-Bold.ttf))
 $(eval $(call create-font-symlink,DroidSerif-Regular.ttf,NotoSerif-Regular.ttf))
 $(eval $(call create-font-symlink,DroidSerif-Bold.ttf,NotoSerif-Bold.ttf))
 $(eval $(call create-font-symlink,DroidSerif-Italic.ttf,NotoSerif-Italic.ttf))
diff --git a/data/fonts/fonts.xml b/data/fonts/fonts.xml
index 0b027485f79..57aca097a1c 100644
--- a/data/fonts/fonts.xml
+++ b/data/fonts/fonts.xml
@@ -22,26 +22,28 @@
 <familyset version="23">
     <!-- first font is default -->
     <family name="sans-serif">
-        <font weight="100" style="normal">GoogleSans-Thin.ttf</font>
-        <font weight="100" style="italic">GoogleSans-ThinItalic.ttf</font>
-        <font weight="300" style="normal">GoogleSans-Light.ttf</font>
-        <font weight="300" style="italic">GoogleSans-LightItalic.ttf</font>
-        <font weight="400" style="normal">GoogleSans-Regular.ttf</font>
-        <font weight="400" style="italic">GoogleSans-Italic.ttf</font>
-        <font weight="500" style="normal">GoogleSans-Medium.ttf</font>
-        <font weight="500" style="italic">GoogleSans-MediumItalic.ttf</font>
-        <font weight="900" style="normal">GoogleSans-Black.ttf</font>
-        <font weight="900" style="italic">GoogleSans-BlackItalic.ttf</font>
-        <font weight="700" style="normal">GoogleSans-Bold.ttf</font>
-        <font weight="700" style="italic">GoogleSans-BoldItalic.ttf</font>
+        <font weight="100" style="normal">Roboto-Thin.ttf</font>
+        <font weight="100" style="italic">Roboto-ThinItalic.ttf</font>
+        <font weight="300" style="normal">Roboto-Light.ttf</font>
+        <font weight="300" style="italic">Roboto-LightItalic.ttf</font>
+        <font weight="400" style="normal">Roboto-Regular.ttf</font>
+        <font weight="400" style="italic">Roboto-Italic.ttf</font>
+        <font weight="500" style="normal">Roboto-Medium.ttf</font>
+        <font weight="500" style="italic">Roboto-MediumItalic.ttf</font>
+        <font weight="900" style="normal">Roboto-Black.ttf</font>
+        <font weight="900" style="italic">Roboto-BlackItalic.ttf</font>
+        <font weight="700" style="normal">Roboto-Bold.ttf</font>
+        <font weight="700" style="italic">Roboto-BoldItalic.ttf</font>
     </family>
 
     <!-- Note that aliases must come after the fonts they reference. -->
     <alias name="sans-serif-thin" to="sans-serif" weight="100" />
     <alias name="sans-serif-light" to="sans-serif" weight="300" />
     <alias name="sans-serif-medium" to="sans-serif" weight="500" />
+    <alias name="google-sans-medium" to="sans-serif" weight="500" />
     <alias name="sans-serif-black" to="sans-serif" weight="900" />
     <alias name="arial" to="sans-serif" />
+    <alias name="roboto" to="sans-serif" />
     <alias name="google-sans" to="sans-serif" />
     <alias name="helvetica" to="sans-serif" />
     <alias name="tahoma" to="sans-serif" />
@@ -49,28 +51,18 @@
     <alias name="verdana" to="sans-serif" />
 
     <family name="sans-serif-condensed">
-        <font weight="300" style="normal">GoogleSansCondensed-Light.ttf</font>
-        <font weight="300" style="italic">GoogleSansCondensed-LightItalic.ttf</font>
-        <font weight="400" style="normal">GoogleSansCondensed-Regular.ttf</font>
-        <font weight="400" style="italic">GoogleSansCondensed-Italic.ttf</font>
-        <font weight="500" style="normal">GoogleSansCondensed-Medium.ttf</font>
-        <font weight="500" style="italic">GoogleSansCondensed-MediumItalic.ttf</font>
-        <font weight="700" style="normal">GoogleSansCondensed-Bold.ttf</font>
-        <font weight="700" style="italic">GoogleSansCondensed-BoldItalic.ttf</font>
+        <font weight="300" style="normal">RobotoCondensed-Light.ttf</font>
+        <font weight="300" style="italic">RobotoCondensed-LightItalic.ttf</font>
+        <font weight="400" style="normal">RobotoCondensed-Regular.ttf</font>
+        <font weight="400" style="italic">RobotoCondensed-Italic.ttf</font>
+        <font weight="500" style="normal">RobotoCondensed-Medium.ttf</font>
+        <font weight="500" style="italic">RobotoCondensed-MediumItalic.ttf</font>
+        <font weight="700" style="normal">RobotoCondensed-Bold.ttf</font>
+        <font weight="700" style="italic">RobotoCondensed-BoldItalic.ttf</font>
     </family>
     <alias name="sans-serif-condensed-light" to="sans-serif-condensed" weight="300" />
     <alias name="sans-serif-condensed-medium" to="sans-serif-condensed" weight="500" />
 
-    <family name="roboto-mini">
-        <font weight="400" style="normal">RobotoMini-Regular.ttf</font>
-        <font weight="400" style="italic">RobotoMini-Italic.ttf</font>
-        <font weight="500" style="normal">RobotoMini-Medium.ttf</font>
-        <font weight="500" style="italic">RobotoMini-MediumItalic.ttf</font>
-        <font weight="700" style="normal">RobotoMini-Bold.ttf</font>
-        <font weight="700" style="italic">RobotoMini-BoldItalic.ttf</font>
-    </family>
-    <alias name="roboto-mini-bold" to="roboto-mini" weight="700" />
-
     <family name="serif">
         <font weight="400" style="normal">NotoSerif-Regular.ttf</font>
         <font weight="700" style="normal">NotoSerif-Bold.ttf</font>
@@ -117,8 +109,6 @@
         <font weight="400" style="italic">SourceSansPro-Italic.ttf</font>
         <font weight="600" style="normal">SourceSansPro-SemiBold.ttf</font>
         <font weight="600" style="italic">SourceSansPro-SemiBoldItalic.ttf</font>
-        <font weight="700" style="normal">SourceSansPro-Bold.ttf</font>
-        <font weight="700" style="italic">SourceSansPro-BoldItalic.ttf</font>
     </family>
     <alias name="source-sans-pro-semi-bold" to="source-sans-pro" weight="600" />
 
@@ -149,8 +139,6 @@
         <font weight="400" style="italic">Rubik-Italic.ttf</font>
         <font weight="500" style="normal">Rubik-Medium.ttf</font>
         <font weight="500" style="italic">Rubik-MediumItalic.ttf</font>
-        <font weight="700" style="normal">Rubik-Bold.ttf</font>
-        <font weight="700" style="italic">Rubik-BoldItalic.ttf</font>
     </family>
     <alias name="rubik-medium" to="rubik" weight="500" />
 
@@ -163,17 +151,6 @@
     <alias name="zilla-slab-bold" to="zilla-slab" weight="700" />
     <alias name="zilla-slab-medium" to="zilla-slab" weight="600" />
 
-    <family name="simple-day">
-        <font weight="400" style="normal">SimpleDay-Regular.ttf</font>
-        <font weight="400" style="italic">SimpleDay-Italic.ttf</font>
-        <font weight="500" style="normal">SimpleDay-Medium.ttf</font>
-        <font weight="500" style="italic">SimpleDay-MediumItalic.ttf</font>
-        <font weight="700" style="normal">SimpleDay-Bold.ttf</font>
-        <font weight="700" style="italic">SimpleDay-BoldItalic.ttf</font>
-    </family>
-    <alias name="simple-day-bold" to="simple-day" weight="700" />
-    <alias name="simple-day-medium" to="simple-day" weight="500" />
-
     <family name="arsenal">
         <font weight="400" style="normal">Arsenal-Regular.ttf</font>
         <font weight="400" style="italic">Arsenal-Italic.ttf</font>
@@ -185,8 +162,6 @@
     <family name="ubuntu">
         <font weight="400" style="normal">Ubuntu-Regular.ttf</font>
         <font weight="400" style="italic">Ubuntu-Italic.ttf</font>
-        <font weight="500" style="normal">Ubuntu-Medium.ttf</font>
-        <font weight="500" style="italic">Ubuntu-MediumItalic.ttf</font>
         <font weight="700" style="normal">Ubuntu-Medium.ttf</font>
         <font weight="700" style="italic">Ubuntu-MediumItalic.ttf</font>
     </family>
@@ -195,8 +170,6 @@
     <family name="alegreya-sans">
         <font weight="400" style="normal">AlegreyaSans-Regular.ttf</font>
         <font weight="400" style="italic">AlegreyaSans-Italic.ttf</font>
-        <font weight="500" style="normal">AlegreyaSans-Medium.ttf</font>
-        <font weight="500" style="italic">AlegreyaSans-MediumItalic.ttf</font>
         <font weight="700" style="normal">AlegreyaSans-Medium.ttf</font>
         <font weight="700" style="italic">AlegreyaSans-MediumItalic.ttf</font>
     </family>
diff --git a/packages/overlays/Android.mk b/packages/overlays/Android.mk
index a4a93e07bba..ce923c40016 100644
--- a/packages/overlays/Android.mk
+++ b/packages/overlays/Android.mk
@@ -53,14 +53,13 @@ LOCAL_REQUIRED_MODULES := \
 	FontLatoOverlay \
 	FontLobsterOverlay \
 	FontMontserratOverlay \
+	FontNotoSansOverlay \
 	FontNotoSerifOverlay \
 	FontOswaldOverlay \
 	FontPacificoOverlay \
 	FontPhilosopherOverlay \
-	FontRobotoOverlay \
 	FontRubikOverlay \
 	FontSourceOverlay \
-	FontSimpledayOverlay \
 	FontUbuntuOverlay \
 	FontViaodalibreOverlay \
 	FontVollkornOverlay \
diff --git a/packages/overlays/FontRobotoOverlay/Android.mk b/packages/overlays/FontRobotoOverlay/Android.mk
deleted file mode 100644
index 805be23da29..00000000000
--- a/packages/overlays/FontRobotoOverlay/Android.mk
+++ /dev/null
@@ -1,31 +0,0 @@
-#
-#  Copyright 2019, The Android Open Source Project
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_RRO_THEME := FontRoboto
-
-LOCAL_PRODUCT_MODULE := true
-
-LOCAL_SRC_FILES := $(call all-subdir-java-files)
-
-LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
-
-LOCAL_PACKAGE_NAME := FontRobotoOverlay
-LOCAL_SDK_VERSION := current
-
-include $(BUILD_RRO_PACKAGE)
diff --git a/packages/overlays/FontRobotoOverlay/AndroidManifest.xml b/packages/overlays/FontRobotoOverlay/AndroidManifest.xml
deleted file mode 100644
index c11ccfb4cfe..00000000000
--- a/packages/overlays/FontRobotoOverlay/AndroidManifest.xml
+++ /dev/null
@@ -1,31 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.android.theme.font.roboto"
-    android:versionCode="1"
-    android:versionName="1.0">
-    <overlay android:targetPackage="android"
-        android:category="android.theme.customization.font"
-        android:priority="1"/>
-
-    <application android:label="@string/font_roboto_overlay" android:hasCode="false">
-        <meta-data
-            android:name="android.theme.customization.REQUIRED_SYSTEM_FONTS"
-            android:value="roboto-mini,roboto-mini-bold" />
-    </application>
-</manifest>
diff --git a/packages/overlays/FontRobotoOverlay/res/values/config.xml b/packages/overlays/FontRobotoOverlay/res/values/config.xml
deleted file mode 100644
index b9e6e6d6739..00000000000
--- a/packages/overlays/FontRobotoOverlay/res/values/config.xml
+++ /dev/null
@@ -1,27 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-     <!-- Name of a font family to use for body text. -->
-    <string name="config_bodyFontFamily" translatable="false">roboto-mini</string>
-    <!-- Name of a font family to use for medium body text. -->
-    <string name="config_bodyFontFamilyMedium" translatable="false">roboto-mini-bold</string>
-    <!-- Name of a font family to use for headlines. If empty, falls back to platform default -->
-    <string name="config_headlineFontFamily" translatable="false">roboto-mini</string>
-    <!-- Name of the font family used for system surfaces where the font should use medium weight -->
-    <string name="config_headlineFontFamilyMedium" translatable="false">roboto-mini-bold</string>
-</resources>
diff --git a/packages/overlays/FontRobotoOverlay/res/values/strings.xml b/packages/overlays/FontRobotoOverlay/res/values/strings.xml
deleted file mode 100644
index e35613d1bef..00000000000
--- a/packages/overlays/FontRobotoOverlay/res/values/strings.xml
+++ /dev/null
@@ -1,20 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="font_roboto_overlay" translatable="false">Roboto</string>
-</resources>
diff --git a/packages/overlays/FontSimpleDayOverlay/Android.mk b/packages/overlays/FontSimpleDayOverlay/Android.mk
deleted file mode 100644
index b498d181601..00000000000
--- a/packages/overlays/FontSimpleDayOverlay/Android.mk
+++ /dev/null
@@ -1,31 +0,0 @@
-#
-#  Copyright 2019, The Android Open Source Project
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_RRO_THEME := FontSimpleDay
-
-LOCAL_PRODUCT_MODULE := true
-
-LOCAL_SRC_FILES := $(call all-subdir-java-files)
-
-LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
-
-LOCAL_PACKAGE_NAME := FontSimpleDayOverlay
-LOCAL_SDK_VERSION := current
-
-include $(BUILD_RRO_PACKAGE)
diff --git a/packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml b/packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml
deleted file mode 100644
index 2d7c111ea94..00000000000
--- a/packages/overlays/FontSimpleDayOverlay/AndroidManifest.xml
+++ /dev/null
@@ -1,31 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.android.theme.font.simpleday"
-    android:versionCode="1"
-    android:versionName="1.0">
-    <overlay android:targetPackage="android"
-        android:category="android.theme.customization.font"
-        android:priority="1"/>
-
-    <application android:label="@string/font_simpleday_overlay" android:hasCode="false">
-        <meta-data
-            android:name="android.theme.customization.REQUIRED_SYSTEM_FONTS"
-            android:value="simple-day,simple-day-bold" />
-    </application>
-</manifest>
diff --git a/packages/overlays/FontSimpleDayOverlay/res/values/config.xml b/packages/overlays/FontSimpleDayOverlay/res/values/config.xml
deleted file mode 100644
index 0ad50d70d8c..00000000000
--- a/packages/overlays/FontSimpleDayOverlay/res/values/config.xml
+++ /dev/null
@@ -1,27 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-     <!-- Name of a font family to use for body text. -->
-    <string name="config_bodyFontFamily" translatable="false">simple-day</string>
-    <!-- Name of a font family to use for medium body text. -->
-    <string name="config_bodyFontFamilyMedium" translatable="false">simple-day-bold</string>
-    <!-- Name of a font family to use for headlines. If empty, falls back to platform default -->
-    <string name="config_headlineFontFamily" translatable="false">simple-day</string>
-    <!-- Name of the font family used for system surfaces where the font should use medium weight -->
-    <string name="config_headlineFontFamilyMedium" translatable="false">simple-day-bold</string>
-</resources>
diff --git a/packages/overlays/FontSimpleDayOverlay/res/values/strings.xml b/packages/overlays/FontSimpleDayOverlay/res/values/strings.xml
deleted file mode 100644
index fa5aabcddfe..00000000000
--- a/packages/overlays/FontSimpleDayOverlay/res/values/strings.xml
+++ /dev/null
@@ -1,20 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="font_simpleday_overlay" translatable="false">Simple Day</string>
-</resources>
diff --git a/packages/overlays/FontSimpledayOverlay/Android.mk b/packages/overlays/FontSimpledayOverlay/Android.mk
deleted file mode 100644
index 5ca63aac3a6..00000000000
--- a/packages/overlays/FontSimpledayOverlay/Android.mk
+++ /dev/null
@@ -1,31 +0,0 @@
-#
-#  Copyright 2019, The Android Open Source Project
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-
-LOCAL_PATH:= $(call my-dir)
-include $(CLEAR_VARS)
-
-LOCAL_RRO_THEME := FontSimpleday
-
-LOCAL_PRODUCT_MODULE := true
-
-LOCAL_SRC_FILES := $(call all-subdir-java-files)
-
-LOCAL_RESOURCE_DIR := $(LOCAL_PATH)/res
-
-LOCAL_PACKAGE_NAME := FontSimpledayOverlay
-LOCAL_SDK_VERSION := current
-
-include $(BUILD_RRO_PACKAGE)
diff --git a/packages/overlays/FontSimpledayOverlay/AndroidManifest.xml b/packages/overlays/FontSimpledayOverlay/AndroidManifest.xml
deleted file mode 100644
index 81fc18c2d28..00000000000
--- a/packages/overlays/FontSimpledayOverlay/AndroidManifest.xml
+++ /dev/null
@@ -1,31 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.android.theme.font.simpleday"
-    android:versionCode="1"
-    android:versionName="1.0">
-    <overlay android:targetPackage="android"
-        android:category="android.theme.customization.font"
-        android:priority="1"/>
-
-    <application android:label="@string/font_simpleday_overlay" android:hasCode="false">
-        <meta-data
-            android:name="android.theme.customization.REQUIRED_SYSTEM_FONTS"
-            android:value="simpleday,simpleday-bold" />
-    </application>
-</manifest>
diff --git a/packages/overlays/FontSimpledayOverlay/res/values/config.xml b/packages/overlays/FontSimpledayOverlay/res/values/config.xml
deleted file mode 100644
index cbd1cf07eb7..00000000000
--- a/packages/overlays/FontSimpledayOverlay/res/values/config.xml
+++ /dev/null
@@ -1,27 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-     <!-- Name of a font family to use for body text. -->
-    <string name="config_bodyFontFamily" translatable="false">simpleday</string>
-    <!-- Name of a font family to use for medium body text. -->
-    <string name="config_bodyFontFamilyMedium" translatable="false">simpleday-bold</string>
-    <!-- Name of a font family to use for headlines. If empty, falls back to platform default -->
-    <string name="config_headlineFontFamily" translatable="false">simpleday</string>
-    <!-- Name of the font family used for system surfaces where the font should use medium weight -->
-    <string name="config_headlineFontFamilyMedium" translatable="false">simpleday-bold</string>
-</resources>
diff --git a/packages/overlays/FontSimpledayOverlay/res/values/strings.xml b/packages/overlays/FontSimpledayOverlay/res/values/strings.xml
deleted file mode 100644
index fa5aabcddfe..00000000000
--- a/packages/overlays/FontSimpledayOverlay/res/values/strings.xml
+++ /dev/null
@@ -1,20 +0,0 @@
-<!--
-/**
- * Copyright (c) 2019, The Android Open Source Project
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
--->
-<resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="font_simpleday_overlay" translatable="false">Simple Day</string>
-</resources>
-- 
2.17.1

